src/main/java/core/basesyntax/suppliers/ColorSupplier.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.suppliers;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static final String DEFAULT_COLOR = "white";
+    public static final String[] COLORS = {
+            DEFAULT_COLOR,
+            "black",
+            "red",
+            "orange",
+            "grey",
+            "brown",
+            "lime",
+            "violet"
+    };
```

Comment:
it's better to make enum in a separate file 
also, all enums have their convention, you should write it like BLACK, RED , e.t.c